src/PowerShellEditorServices/Services/PowerShell/Handlers/ExpandAliasHandler.cs

```diff
@@ -63,13 +61,15 @@ public async Task<ExpandAliasResult> Handle(ExpandAliasParams request, Cancellat
 }";
 
             // TODO: Refactor to not rerun the function definition every time.
-            var psCommand = new PSCommand();
-            psCommand
+            PSCommand psCommand = new PSCommand()
```

Comment:
```suggestion
            PSCommand psCommand = new()
```

Reply:
Can't cause we proceed to call methods inline.